fireatlas/FireRunDaskCoordinator.py

```diff
@@ -75,14 +75,21 @@ def job_fire_forward(region: Region, tst: TimeStep, ted: TimeStep):
         allfires, allpixels, t_saved = Fire_Forward(tst=tst, ted=ted, region=region, restart=False)
         copy_from_local_to_s3(allpixels_filepath(tst, ted, region, location="local"), fs)
         copy_from_local_to_s3(allfires_filepath(tst, ted, region, location="local"), fs)
-
         allfires_gdf = allfires.gdf
+        if t_saved is None:
```

Comment:
@jsignell: I made the fix here instead of `FireForward` b/c it only seems to be used here but the question is will this blow up in other places? I think not

Reply:
Actually, I say we ignore `t_saved` coming back here and just pass `tst` to `save_snapshots` and just overwrite. That seems most logical to me. Thoughts?